steps/redis/get/main.go

```diff
@@ -21,43 +24,56 @@ type RedisGet struct {
 	redisClient *redis.Client
 }
 
-func (l *RedisGet) Init() error {
+func (l *RedisGet) Init() *step.Error {
 	err := envconf.Parse(&l.args)
 	if err != nil {
-		return err
+		return &step.Error{Code: step.InvalidArgument, Message: "invalid env vars provided", Verbose: fmt.Errorf("error: %w", err).Error()}
 	}
 
 	redisClient, err := base.InitRedisClient(l.args.Args)
 	if err != nil {
-		return err
+		return &step.Error{Code: step.Unavailable, Message: "failed connect redis client", Verbose: fmt.Errorf("error: %w", err).Error()}
 	}
 
 	l.redisClient = redisClient
 
 	return nil
 }
 
-func (l *RedisGet) Run() (exitCode int, output []byte, err error) {
+func (l *RedisGet) Run() (outputJson step.Output, stepError *step.Error) {
 	gc := gabs.New()
 
 	cmd := l.redisClient.Get(context.Background(), l.args.Key)
 	if cmd == nil {
-		return step.ExitCodeFailure, nil, fmt.Errorf("invalid result returned from get operation")
+		return nil, &step.Error{Code: step.InvalidArgument, Message: "invalid result returned from get operation"}
 	}
 
 	val, err := cmd.Result()
 	if err != nil {
 		if err == redis.Nil {
-			return step.ExitCodeFailure, nil, fmt.Errorf("key not found")
+			return nil, &step.Error{Code: step.NotFound, Message: "key not found"}
 		}
-		return step.ExitCodeFailure, nil, fmt.Errorf("failed getting key value: validate key is valid: %w", err)
+		return nil, &step.Error{Code: step.InvalidArgument, Message: "failed getting key value", Verbose: fmt.Sprintf("validate key is valid: %w", err)}
```

Comment:
```suggestion
		return nil, &step.Error{Code: step.InvalidArgument, Message: "failed getting key value", Verbose: fmt.Sprintf("validate key is valid: %s", err.Error())}
```
Cant use %w outside of fmt.Errorf